twitter_ads/campaign.py

```diff
@@ -247,6 +247,7 @@ class Campaign(Resource, Persistence, Analytics, Batch):
 resource_property(Campaign, 'standard_delivery', transform=TRANSFORM.BOOL)
 resource_property(Campaign, 'start_time', transform=TRANSFORM.TIME)
 resource_property(Campaign, 'total_budget_amount_local_micro')
+resource_property(Campaign, 'with_draft')
```

Comment:
IIRC, this is not needed. Resource properties are for keys in the entity objects.

Reply:
+1 should be okay to remove this